qemu/tests/balloon_check.py

```diff
@@ -107,6 +107,7 @@ def balloon_memory(self, new_mem):
         elif new_mem == 0:
             compare_mem = self.current_mmem
         elif new_mem <= 100:
+            self.wait_for_balloon_complete(self.params.get("balloon_timeout", 240))
```

Comment:
I am not sure does it need to call `wait_for_balloon_complete` everytime before calling `get_ballooned_memory`

Reply:
Yes if new_mem < 100.
And it should do it, because balloon mem from large mem to very small value takes longer time than normal.

Reply:
ok, got it, thanks